### PR TITLE
[LC-896] Add LFT `app` to loopchain

### PR DESCRIPTION
**Pull request by zsaladin**
`app` of LFT is added to loopchain.
The integration with loopchain means modifying the `app` to work well with components of loopchain.

### CODE REVIEW

File: loopchain/launcher.py [hunk 1]
```diff
@@ -116,7 +118,14 @@ def start_as_channel(args):
...
 
-    ChannelService(channel, amqp_target, amqp_key).serve()
+    try:
+        ChannelService(channel, amqp_target, amqp_key).serve()
+    except ConsensusChanged as e:
+        utils.logger.info(f"Consensus Changed")
+        utils.logger.info(f"Remain txs. {len(e.remain_txs)}")
+        utils.logger.info(f"Last unconfirmed block {e.last_unconfirmed_block.header}")
+        utils.logger.info(f"Last unconfirmed votes {e.last_unconfirmed_votes}")
+        Node(e.node_id).start()
```
Comment (yourhrh) [inline loopchain/launcher.py]: I think new node needs all informations of ConsensusChanged exception. Will this pass in other feature?

Comment (zsaladin) [reply loopchain/launcher.py]: Yeah, Purpose of this PR is just for working without any problem with minimum number of modifications from original LFT `app`.

**Review by yourhrh — APPROVED** [review]

File: loopchain/channel/channel_service.py [hunk 2]
```diff
@@ -466,8 +466,8 @@ def __ready_to_height_sync(self):
             if self._is_genesis_node():
                 self.generate_genesis_block()
 
-        if not self.is_support_node_function(conf.NodeFunction.Vote) and not ChannelProperty().rs_target:
-            utils.exit_and_msg(f"There's no radiostation target to sync block.")
+        # if not self.is_support_node_function(conf.NodeFunction.Vote) and not ChannelProperty().rs_target:
+        #     utils.exit_and_msg(f"There's no radiostation target to sync block.")
```
Comment (yakkle) [inline loopchain/channel/channel_service.py]: Why don't you remove if this lines useless? or add short description why you add comment this lines.

File: loopchain/app/ui/console.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import asyncio
+import IPython
```
Comment (yakkle) [inline loopchain/app/ui/console.py]: Is `IPtyhon` library essential on running loopchain?
What is the purpose of the `ui.console`?

Comment (zsaladin) [reply loopchain/app/ui/console.py]: The console is a part of LFT application. It is for easy debugging and operate node's action in runtime.
It is like web3.js or web3.py in ethereum.

It has to be installed in develop mode.
But now we will use LFT's console itself.

File: loopchain/channel/channel_service.py (hunk 2 above)
Comment (zsaladin) [reply loopchain/channel/channel_service.py]: The PR is for operating loopchain as a single node. So RS Target is not required. But current loopchain requires RS Target. 
The reason why I leave it as a commit is We do not decide how we solve it yet.

**Comment by metalg0su** [comment]
retest this please

**Pull request merged**